### DISCUSSION TITLE
Setup Postal With Sub Domain, Can't Receive Email

### DISCUSSION THREAD

**Discussion by ridwankustanto** [postalserver/postal#1748] [Help with using Postal]
## Describe the bug

Hi, 

I'm not sure if it's a bug or not. I can send email but I can't receive email even my DNS configuration is passed. I suspect it because I setup my postal with subdomain on my MX record. 

## To Reproduce

1. Setup postal with subdmain
2. Set MX configuration sub.domain.com sub.domain.com; 10

## Expected behaviour

I want to receive email with my subdomain like: mail@sub.domain.com

## My configuration

I follow all the DNS setup from this [reference](https://github.com/postalserver/postal/issues/301#issuecomment-322665936). But instead of mydomain.com MX 10 mail.mydomain.com, I set it up like mail.mydomain.com MX 10 mail.mydomain.com.

## Environment details 

 - OS: Docker

**Comment by willpower232** [comment]
So when you email `mail@sub.domain.com` what happens? Do you get a bounce back with an error message? Does it appear in Postal at all?

**Reply by ridwankustanto** [reply]
> So when you email `mail@sub.domain.com` what happens? Do you get a bounce back with an error message? Does it appear in Postal at all?

When I tried to send an email with Gmail, the mail was sent, but my postal never get it. I installed it with docker, I check on the worker container, there is no log of that.

Oh, and I tried to send mail from postal "Simulate an incoming e-mail" menu, I get the email but the status is "PROCESSED Message has been accepted but not sent to any endpoints.".

![image](https://user-images.githubusercontent.com/19152005/144213220-f122e262-3f6b-4678-83af-666dd7ba839b.png)

**Reply by dkmailer247** [reply]
any update on it i am facing same issue

**Reply by ridwankustanto** [reply]
Still has this problem. Have someone experienced it before?

**Reply by willpower232** [reply]
Are you able to email this address from something that isn't Gmail?

If you can't send email from other providers then that means its either a firewall blocking port 25 (or your Postal not listening on that port) or your DNS is not correct. Postal can do a good guess at verifying your records are correct but it is possible they are not.
